src/org/usfirst/frc/team1529/robot/commands/HandIntakeCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team1529.robot.commands;
+
+import org.usfirst.frc.team1529.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class HandIntakeCommand extends Command {
+
+    public HandIntakeCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.kHandSubsystem);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	Robot.kHandSubsystem.Intake();
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return true;
```

Comment:
For all of these, you might want to consider when to call it finished. Right now there are two issues that could occur by just setting isFinished directly to true;

- Command is Called, checked is Finished right away, never actually executes
- Command is called, executes, one time, and ends